woocommerce/Lifecycle.php

```diff
@@ -390,18 +390,18 @@ protected function generate_milestone_notice_message( $custom_message ) {
 		if ( $this->get_plugin()->get_reviews_url() ) {
 
 			// to be prepended at random to each milestone notice
-			$exclamations = array(
-				__( 'Awesome', 'woocommerce-plugin-framework' ),
-				__( 'Fantastic', 'woocommerce-plugin-framework' ),
-				__( 'Cowabunga', 'woocommerce-plugin-framework' ),
-				__( 'Congratulations', 'woocommerce-plugin-framework' ),
-				__( 'Hot dog', 'woocommerce-plugin-framework' ),
-			);
+			$exclamations = [
+				_x( 'Awesome!', 'Generic congratulatory exclamation at the start of a milestone message', 'woocommerce-plugin-framework' ),
+				_x( 'Fantastic!', 'Generic congratulatory exclamation at the start of a milestone message', 'woocommerce-plugin-framework' ),
+				_x( 'Cowabunga!', 'Generic congratulatory exclamation at the start of a milestone message', 'woocommerce-plugin-framework' ),
```

Comment:
😂

Reply:
they had a question about "Hot dog"! I LOL'ed -- taken out of context is funny